src/Playwright/build/Microsoft.Playwright.targets

```diff
@@ -21,4 +21,13 @@
     <Message Text="[Playwright] Copying shell script from $(MSBuildThisFileDirectory) to $(OutputPath)..." />
     <Copy SourceFiles="@(_CopyItems)" DestinationFiles="@(_CopyItems->'$(OutDir)\%(RecursiveDir)%(Filename)%(Extension)')"/>
   </Target>
+  <Target Name="PlaywrightCopyAfterPublish" AfterTargets="Publish">
```

Comment:
It feels like we are giving a bit of false hope that this will work for users. It also breaks their existing builds. Should we let them do it themselves?

Reply:
I've tested this and it will work in the Single-file example, because the `.playwright` folder gets copied next to the file, and the `BaseDirectory` path change means the correct root folder is picked up.